packages/browserslist-config/package.json

```diff
@@ -20,7 +20,7 @@
     "access": "public"
   },
   "engines": {
-    "node": "^8 || ^10 || ^12"
+    "node": ">=10"
```

Comment:
Had some issues with `Current` in the past, are we sure the "current" and not up to LTS is usable?

Reply:
Yeah, that's fair. Travis builds against the two current LTS versions (10, 12) + latest/current (13).

The idea is that people can at least try it out against current without hitting error/warnings and we can build/test against the current before the next LTS is launched.